src/session.ts

```diff
@@ -222,18 +239,25 @@ class Session extends ServiceObject {
    * @example
    * session.beginTransaction(function(err, transaction, apiResponse) {});
    */
-  beginTransaction(options, callback) {
-    if (is.fn(options)) {
-      callback = options;
-      options = {};
-    }
+  beginTransaction(options: {}): Promise<BeginTransactionResponse>;
```

Comment:
This should be an interface named `BeginTransactionRequest`.  Here's a link to the parameters:
https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.v1#google.spanner.v1.BeginTransactionRequest

Reply:
👋  @nareshqlogic *gentle ping*

Reply:
@JustinBeckwith , apologize for late reply.
Documentation is not clearly state what property should have inside TransactionOptions interface. So I created interface mentioned description in src/transactions.ts file for Read/write transaction options. Let me know whether it is correct or not.